### DISCUSSION TITLE
Get specific Leaderboard Entry

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#801] [Ideas]
**Background Information:**
RetroAchievement supports leaderboards for many games for their time trial modes, highscore hunt and whatever. One issue is there are only 50 users (51 when the own username is not in Top 50) shown when checking the leaderboard. While I can understand that performance based it wouldn't be good to load over 500+ entries at once I would at least have a way to get more entries somehow.

Leaderboards are often a good way to check if players finished a challenge or not, a good example is this:
http://retroachievements.org/achievement/72255
http://retroachievements.org/leaderboardinfo.php?i=2035

The ingame timer however only shows X'XX''X. Which means when the player finishs the race with a time of 0'17''0(3) it would look like it didn't trigger correctly in the game, while in fact the player didn't complete the challenge. There are many entries way under 17 seconds so there is no possibility to see an entry like that with the current view.

**Task:**
I would like to have a field where I can insert an username and then get only the entry of this user on the leaderboard or, if there is no entry, a message that no entry exists for this user.

**Comment by davehx** [comment]
Seconded. Or maybe an API call for one/all of...
1) Full leaderboard 
2) All leaderboard positions and scores for user xxxx in game yyyyy
3) User xxxxx's position and score on an individual leaderboard

Thanks

**Comment by Jamiras** [comment]
#627 updated the client API to support fetching entries around a user and fetching any subset of entries from the list (N entries starting at X). Each leaderboard still needs to be queried separately. 

The original request seems more like an administrative feature. You normally wouldn't need to get the actual entry for another user.
